### PR TITLE
Add longtable "rerun" message

### PR DESCRIPTION
**Pull request by eheinzen**
See #124

### CODE REVIEW

File: R/latex.R [hunk 1]
```diff
@@ -238,7 +238,7 @@ latexmk_emu = function(
   }
   for (i in seq_len(times)) {
     if (file.exists(logfile)) {
-      if (!any(grepl('(Rerun to get|Please \\(re\\)run) ', readLines(logfile), useBytes = TRUE))) break
+      if (!any(grepl('(Rerun to get|Please \\(re\\)run) | Rerun LaTeX\\.', readLines(logfile), useBytes = TRUE))) break
```
Comment (yihui) [inline R/latex.R]: ` | Rerun LaTeX\\.` needs to be moved into `()`:

```suggestion
      if (!any(grepl('(Rerun to get|Please \\(re\\)run | Rerun LaTeX\\.)', readLines(logfile), useBytes = TRUE))) break
```

Comment (eheinzen) [reply R/latex.R]: In that case, should I add an extra space after "get"?  That would preserve the original regex.

Comment (yihui) [reply R/latex.R]: Oh yes.

**Comment by eheinzen** [comment]
Thanks so much!

**Pull request merged**